templates/README.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Accuracy_metric
+datasets: - some_dataset
+tags: - metric
+sdk: gradio
+sdk_version: 2.8.13
+app_file: app.py
+pinned: false
+---
+
+# Metric
+
+## Metric description
+
+## How to use
+
+## Examples 
+
+## References 
+
+## Limitations and bias
```

Comment:
This won't show up in the space, so I wonder if this should instead be in some place that will later be displayed in the Space. You already do this with `_DESCRIPTION` for example, so I was wondering if all of this should be over there instead.

Reply:
Yes, indeed there is some duplication here. I was thinking we could read the `README.md` in the gradio app and display it. The reasons why I thought it is nicer as a separate file:
- it's easier to edit a markdown file directly than a string in Python
- if we ever decide to make a dedicated metrics/evaluate repository type we would already have a README for all repos

Reply:
Yes, I think that's good. We can read the `md` file content and add it to the article section of the demo.